nikola/plugins/command/import_wordpress.py

```diff
@@ -165,6 +166,12 @@ def show_info_about_mising_module(modulename):
             self.extra_languages)
         self.context['REDIRECTIONS'] = self.configure_redirections(
             self.url_map)
+
+        # Add tag redirects
+        for tag in self.all_tags:
+            tag = utils.slugify(tag.decode('utf8'))
+            self.url_map['{}tag/{}'.format(self.context['SITE_URL'], tag)] = self.context['SITE_URL'] + 'categories/' + tag + '.html'
```

Comment:
But the path is nowhere in the dump that I can see. Worst case, the user can search/replace.